code/src/components/TotalTasks.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+
+import { tasks } from '../reducers/tasks'
+import { DeleteAllButton } from './styled components/TotalTasksStyling'
+
+const TotalTasks = () => {
+    const dispatch = useDispatch()
+
+    const onClickDeleteAll = () => {
+        dispatch(tasks.actions.deleteAllTasks())
+	}
+
+    return (
+            <DeleteAllButton 
+                onClick={onClickDeleteAll}
+            >   
+                Delete all!
+            </DeleteAllButton>
+    )
+}
+
```

Comment:
Nice done with the delete all button and your reducer! I would like to implement this as well!